src/main/groovy/com/smartbear/swagger/Swagger2TestCaseGenerator.groovy

```diff
@@ -0,0 +1,60 @@
+package com.smartbear.swagger
+
+import com.eviware.soapui.impl.rest.RestRequestInterface.HttpMethod
+import com.eviware.soapui.impl.rest.RestResource
+import com.eviware.soapui.impl.wsdl.WsdlProject
+import com.eviware.soapui.impl.wsdl.WsdlTestSuite
+import com.eviware.soapui.impl.wsdl.testcase.WsdlTestCase
+import com.eviware.soapui.impl.wsdl.teststeps.RestTestRequestStep
+import com.eviware.soapui.impl.wsdl.teststeps.registry.RestRequestStepFactory
+import com.eviware.soapui.model.testsuite.TestProperty
+import io.swagger.models.Operation
+import io.swagger.models.Path
+import io.swagger.models.parameters.AbstractSerializableParameter
+
+class Swagger2TestCaseGenerator {
+    static void generateTestCases(WsdlProject project, RestResource resource, Path path) {
+        WsdlTestSuite testSuite = createTestSuiteIfNotPresent(project)
+        WsdlTestCase testCase = testSuite.addNewTestCase("$resource.name-TestCase")
+        resource.restMethodList.each {
```

Comment:
Not sure this is the right approach since some metadata gets lost in the REST API creation - could we iterate over the Swagger model instead (paths/operations) and create TestSteps directly from it?